config/settings.yml

```
@@ -0,0 +1 @@
+GOOGLE_ANALYTICS_ID:
```

```suggestion
GOOGLE_ANALYTICS_ID: UA-7219229-36
```

I think it's probably fine to put this into the global settings.

---

By default that would add analytics on all our environments (dev, uat) to the same spot (i think)